service/provider/src/main/java/gov/va/vro/service/provider/bip/service/BipApiService.java

```diff
@@ -256,14 +252,7 @@ public BipFileUploadResp uploadEvidenceFile(
       updr.documentTypeId(131);
 
       String filename = file.getOriginalFilename();
-
-      Payload payload = new Payload();
```

Comment:
@wlin2019 I don't see this Payload used anywhere. The same for "UploadRequest", so I deleted them. Should they be used?

Reply:
@dimitri-amida please ignore anything in bipevidence. I put them based on the open api spec I had but since then testing in DEV showed the original in bip were almost correct. I fixed a few fields and they are now correct. I will remove bipevidence so it does not hurt if you remove them here.

Reply:
OK, I deleted this segment of unused code